packages/bar/src/types.ts

```diff
@@ -124,6 +124,7 @@ interface BarCustomLayerBaseProps<RawDatum>
 
     xScale: AnyScale
     yScale: AnyScale
+    getColor: OrdinalColorScaleConfig<ComputedDatum<RawDatum>>
```

Comment:
The type is `OrdinalColorScale<ComputedDatum<RawDatum>>` at this point it's not the config anymore.